esmf_regrid/tests/unit/experimental/unstructured_scheme/test__mesh_to_MeshInfo.py

```diff
@@ -72,6 +72,65 @@ def _example_mesh():
     return mesh
 
 
+def _gridlike_mesh(n_lons, n_lats):
```

Comment:
A small comment here for the unintiated reader would be helpful, especially the indexing / roll operations, to explain what you are creating